### PR TITLE
as an admin i should be able to hide posts 

### PR DESCRIPTION
**Pull request by abdelrahmanMaged**
@Tharwat7 
@BeshoSmile 
@HebaElSobky 

Please review

### CODE REVIEW

File: project/tager/tager_www/templates/ViewPost.html [hunk 1]
```diff
@@ -293,7 +296,7 @@
 
       
 
-    
+ {% if not admin %}
```
Comment (BeshoyAtef) [inline project/tager/tager_www/templates/ViewPost.html:299]: why this is for if not admin ?

Comment (abdelrahmanMaged) [inline project/tager/tager_www/templates/ViewPost.html:299]: as admin will not report the post or add to wishlist or anything it is an
assumption btw if the company object i can remove it

File: project/tager/tager_www/views.py [hunk 2]
```diff
@@ -296,9 +305,9 @@ def add_to_wish_list(request):
 
 def view_post(request):
     user = request.user
-    print "i am form 1yyyyyy"
+    # print "i am form 1yyyyyy"
```
Comment (BeshoyAtef) [inline project/tager/tager_www/views.py:308]: ?

File: project/tager/tager_www/views.py [hunk 3]
```diff
@@ -520,12 +521,11 @@ def UserRegistration(request):
...
 @login_required
 def update_status(request):
+    print 'testing this method'
     user = request.user
-    print user
     user.status = request.POST['status']
-    print request.POST['status']
     user.save()
-    return render(request, 'profile.html', {'user':user})
+    return HttpResponse(" ")
```
Comment (BeshoyAtef) [inline project/tager/tager_www/views.py:528]: ???

**Comment by BeshoyAtef** [comment]
i can see you removed a huge part of code 
any  reason walla jst by mistake ?

**Comment by abdelrahmanMaged** [comment]
i just removed two print statements :O anything else will be by mistake tb3n

**Comment by HebaElSobky** [comment]
Ready to be reviewed or not yet.. T5alas ma3 Beshoy first ??!

**Comment by abdelrahmanMaged** [comment]
i guess you can review the scenario while he is reviewing code :D

**Comment by HebaElSobky** [comment]
Okay I will review within 15min isA

**Comment by abdelrahmanMaged** [comment]
sure take your time

**Comment by HalaKamal** [comment]
@AbdelrahmanAyad @abdelrahmanMaged  by the way i'm your reviewer of uml diagram i'm not supposed to look after you that you uploaded your things to review it u must tag me 
regards

**Comment by HalaKamal** [comment]
the um is wromg reopen it the viewpost html must be in templates @AbdelrahmanAyad @abdelrahmanMaged

**Pull request closed without merging**